examples/basics/plotting/quick.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
+Example of plotting multiple things using the quick interface.
+"""
+
+import numpy as np
+
+from vispy.io import load_data_file
+import vispy.plot.q as vpq
```

Comment:
I am -1 on inheriting baggage from MATLAB. This is a fairly simple API issue that will take most users only a couple of minutes to adjust to. I also don't think having a separate namespace in any way addresses that issue. These users will be equally confused whether they write `vp.plot()` or `vpq.plot()`. They'll look at the examples, and then they'll say "AH! That makes so much more sense than Matplotlib".